unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationBadge/UnreadNotificationBadge.cs

```diff
@@ -0,0 +1,102 @@
+using DCL.Interface;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+
+/// <summary>
+/// Shows the number of unread messages from a friend.
+/// </summary>
+public class UnreadNotificationBadge : MonoBehaviour
+{
+    public TextMeshProUGUI notificationText;
+    public GameObject notificationContainer;
+    public int maxNumberToShow = 9;
+
+    private IChatController currentChatController;
+    private string currentUserId;
+    private long currentTimestampReading;
+    private int currentUnreadMessages;
+
+    public int CurrentUnreadMessages
```

Comment:
Generally for this pattern, we use `variableNameValue` as the backing field and `variableName` as the property. 

For consistency, this variable should be named `currentUnreadMessages` and the backing field `currentUnreadMessagesValue`

Reply:
Great, thanks!